test/controllers/partners_controller_test.rb

```diff
@@ -30,6 +35,7 @@ class PartnersControllerTest < ActionDispatch::IntegrationTest
   test 'should get index without subdomain' do
     get url_for controller: "partners", subdomain: false
     assert_response :success
+
```

Comment:
None of this is interesting. Just some formatting changes while I was debugging the cause of a test failure.